pkg/controller/direct/monitoring/roundtrip_test.go

```diff
@@ -58,33 +57,17 @@ func FuzzMonitoringDashboardSpec(f *testing.F) {
 			".row_layout.rows[].widgets[]",
 		}
 		for _, widgetPath := range widgetPaths {
-			unimplementedFields.Insert(widgetPath + ".xy_chart.data_sets[].target_axis")
-			unimplementedFields.Insert(widgetPath + ".xy_chart.data_sets[].time_series_query.prometheus_query")
-			unimplementedFields.Insert(widgetPath + ".xy_chart.y2_axis")
-			unimplementedFields.Insert(widgetPath + ".xy_chart.timeshift_duration")
-			unimplementedFields.Insert(widgetPath + ".xy_chart.thresholds[].target_axis")
-
-			// This one might be a bug?
-			unimplementedFields.Insert(widgetPath + ".xy_chart.data_sets[].min_alignment_period.nanos")
-
-			unimplementedFields.Insert(widgetPath + ".scorecard.thresholds[].target_axis")
 			unimplementedFields.Insert(widgetPath + ".scorecard.blank_view")
 
-			unimplementedFields.Insert(widgetPath + ".alert_chart")
-
-			unimplementedFields.Insert(widgetPath + ".time_series_table")
-
-			unimplementedFields.Insert(widgetPath + ".pie_chart")
-
-			unimplementedFields.Insert(widgetPath + ".single_view_group")
+			unimplementedFields.Insert(widgetPath + ".pie_chart.data_sets[].time_series_query.time_series_filter.statistical_time_series_filter")
+			unimplementedFields.Insert(widgetPath + ".pie_chart.data_sets[].time_series_query.time_series_filter.pick_time_series_filter.interval")
+			unimplementedFields.Insert(widgetPath + ".pie_chart.data_sets[].time_series_query.time_series_filter_ratio.statistical_time_series_filter")
+			unimplementedFields.Insert(widgetPath + ".pie_chart.data_sets[].time_series_query.time_series_filter_ratio.pick_time_series_filter.interval")
 
-			unimplementedFields.Insert(widgetPath + ".time_series_table")
-
-			unimplementedFields.Insert(widgetPath + ".error_reporting_panel")
-
-			unimplementedFields.Insert(widgetPath + ".incident_list")
-
-			unimplementedFields.Insert(widgetPath + ".id")
+			unimplementedFields.Insert(widgetPath + ".time_series_table.data_sets[].time_series_query.time_series_filter.statistical_time_series_filter")
+			unimplementedFields.Insert(widgetPath + ".time_series_table.data_sets[].time_series_query.time_series_filter.pick_time_series_filter.interval")
```

Comment:
I haven't found any examples for statisticial_time_series_filter or interval, so I'm not sure if we should try to support it yet.